plover/misc.py

```diff
@@ -65,3 +65,18 @@ def boolean(value):
             return False
         raise ValueError(value)
     return bool(value)
+
+
+def surrogate_pair(char):
```

Comment:
Since this function isn't used apart from in to_surrogate_pair, could you inline it?

```python
code_point = ord(code)
if code_point >= 0x10000:
            high = (code_point - 0x10000) // 0x400 + 0xD800
            low = (code_point - 0x10000) % 0x400 + 0xDC00
            pairs += (high, low)
        else:
            pairs.append(code_point)
```